docs/reports/requirements/project_charter.md

```diff
@@ -0,0 +1,181 @@
+        <td>September 23, 2022</td>
+    </tr>
+    <tr>
+        <td>4</td>
+        <td>Requirements gathering, project outlining</td>
+        <td>September 30, 2022</td>
+    </tr>
+    <tr>
+        <td>5</td>
+        <td>Identify tech stack, solidify milestone features</td>
+        <td>October 7, 2022</td>
+    </tr>
+    <tr>
+        <td>6</td>
+        <td>Requirements presentation slides and video, group evaluation</td>
+        <td>October 11* & 14, 2022</td>
+    </tr>
+    <tr>
+        <td>7</td>
+        <td>Finalize requirements, requirements report (PDF), peer evaluation</td>
+        <td>October 21, 2022</td>
+    </tr>
+    <tr>
+        <td>8</td>
+        <td>Complete required tasks</td>
+        <td>October 28, 2022</td>
+    </tr>
+    <tr>
+        <td>9</td>
+        <td>Complete required tasks</td>
+        <td>November 4, 2022</td>
+    </tr>
+    <tr>
+        <td>10</td>
+        <td>Reading week</td>
+        <td>No milestone</td>
+    </tr>
+    <tr>
+        <td>11</td>
+        <td>Create heuristic evaluation, prototype video demo</td>
+        <td>November 18, 2022</td>
+    </tr>
+    <tr>
+        <td>12</td>
+        <td>Peer testing report</td>
+        <td>November 25, 2022</td>
+    </tr>
+    <tr>
+        <td>13</td>
+        <td>Plan for next term</td>
+        <td>December 2, 2022</td>
+</table>
+
+## Term 1 Milestones
+
+1. Requirements report + project requirements presentation (Oct. 11, 14, and 21)
+2. Peer testing report + prototype video demo (Nov. 18)
+
+## Assumptions
+
+* The add-on will receive 1+ audio files in a variety of languages from the existing Kukarella web application
+* The audio produced might not contain accurate pronunciations
+* Kukarella will continue to be in business throughout this project
+
+## Constraints & Risks
+
+* 8 months; limited time to complete the project
+* 4 student developers; limited human resource working on this project
+* Developers have limited experience with the following list, so there is a chance that we've underestimated the work involved.
+    * TypeScript, Node.js
+    * animation
+    * audio and animation synchronization via programming
+* A team member is scheduled for surgery in Term 1, so there will be a drop in production around `[date]`
+
+## Questions for Group and Client
+
+The following questions will help define requirements and allow for the completion of this Project Charter
+
+* Which avatar(s) will we be working with? Which one(s) do(es) the client want to become 'premium'?
+* Where do we get the assets for the avatar(s)?
+* Where do we get the assets for the mouths/lips?
+* How will we measure the success of "realistic lip-syncing"? - Definition: is 'realistic' about the <u>quality</u> of the mouth assets matching the sounds?
+* How will we measure the success for lip-sync timings? - Definition: do the assets line up with when the sounds are made and what is an acceptable marginf for error?
```

Comment:
They aren't important at this point, but when we get to the peer testing part, then it will require some way for our peers to test the product. Just thinking ahead :)